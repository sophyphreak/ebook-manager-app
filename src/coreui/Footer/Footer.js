import React, { Component } from 'react';

class Footer extends Component {
  render() {
    return (
      <footer className="app-footer">
        <span>
          <a href="http://wisefoxpub.com/">Wise Fox</a> &copy; 2018 Redbeard.
        </span>
        <span className="ml-auto">
          Powered by <a href="http://wisefoxpub.com/">Wise Fox!</a>
        </span>
      </footer>
    );
  }
}

export default Footer;
