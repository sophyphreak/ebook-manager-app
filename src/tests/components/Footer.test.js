import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer/Footer';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Footer />);
});

test('should render Footer correctly', () => {
  expect(wrapper).toMatchSnapshot();
});