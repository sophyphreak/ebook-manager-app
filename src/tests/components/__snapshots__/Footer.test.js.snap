// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span>
          <a
            href="http://coreui.io"
          >
            CoreUI
          </a>
           © 2018 creativeLabs.
        </span>,
        <span
          className="ml-auto"
        >
          Powered by 
          <a
            href="http://coreui.io"
          >
            CoreUI
          </a>
        </span>,
      ],
      "className": "app-footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <a
              href="http://coreui.io"
            >
              CoreUI
            </a>,
            " © 2018 creativeLabs.",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "CoreUI",
              "href": "http://coreui.io",
            },
            "ref": null,
            "rendered": "CoreUI",
            "type": "a",
          },
          " © 2018 creativeLabs.",
        ],
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Powered by ",
            <a
              href="http://coreui.io"
            >
              CoreUI
            </a>,
          ],
          "className": "ml-auto",
        },
        "ref": null,
        "rendered": Array [
          "Powered by ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "CoreUI",
              "href": "http://coreui.io",
            },
            "ref": null,
            "rendered": "CoreUI",
            "type": "a",
          },
        ],
        "type": "span",
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span>
            <a
              href="http://coreui.io"
            >
              CoreUI
            </a>
             © 2018 creativeLabs.
          </span>,
          <span
            className="ml-auto"
          >
            Powered by 
            <a
              href="http://coreui.io"
            >
              CoreUI
            </a>
          </span>,
        ],
        "className": "app-footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <a
                href="http://coreui.io"
              >
                CoreUI
              </a>,
              " © 2018 creativeLabs.",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "CoreUI",
                "href": "http://coreui.io",
              },
              "ref": null,
              "rendered": "CoreUI",
              "type": "a",
            },
            " © 2018 creativeLabs.",
          ],
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Powered by ",
              <a
                href="http://coreui.io"
              >
                CoreUI
              </a>,
            ],
            "className": "ml-auto",
          },
          "ref": null,
          "rendered": Array [
            "Powered by ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "CoreUI",
                "href": "http://coreui.io",
              },
              "ref": null,
              "rendered": "CoreUI",
              "type": "a",
            },
          ],
          "type": "span",
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
